README.md

```diff
@@ -1,2 +1,22 @@
-# fuelup
-The Fuel toolchain installer
+# fuelup: the Fuel toolchain installer
+
+_fuelup_ installs the Fuel toolchain from our official release channels, enabling you to easily keep the toolchain updated.
+
+## Installation
+
+Installation is simple: all you need is `fuelup-init.sh`, which downloads the core Fuel binaries needed to get you started on development.
+
+```sh
+curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/FuelLabs/fuelup/master/fuelup-init.sh | sh -s install
+```
+
+This will install `forc`, `forc-fmt`, `forc-explore`, `forc-lsp` as well as `fuel-core` in `~/.fuelup/bin`. You will have to add `~/.fuelup/bin` to your `PATH`.
```

Comment:
Add an explanation for how to do this for various shells. I think `rustup` gives at least one instruction for bash.

Reply:
Wouldn't this one-liner work as well for `bash`? Since `bash` is invoked as `sh` if you run it in a bash system?

this is mostly following what [`rustup`](https://rustup.rs/) does as well

Reply:
Oh my bad, I meant explain "You will have to add `~/.fuelup/bin` to your `PATH`."